arbitrum-docs/launch-orbit-chain/how-tos/orbit-sdk-deploying-anytrust-chain.md

```diff
@@ -24,28 +26,29 @@ See the ["set-valid-keyset" example](https://github.com/OffchainLabs/arbitrum-or
 
 ### About AnyTrust Orbit
 
-AnyTrust chains follow a different model than Rollup chains, offering unique features and deployment processes. For a comprehensive understanding of the general principles of Orbit chains and the differences between Orbit chain types, please refer to the [Orbit SDK introduction](../orbit-sdk-introduction.md) page.
+AnyTrust chains follow the Rollup protocol but offer different Data model availability, features, and deployment processes. For an overview of Orbit chain types, please refer to the [Orbit SDK introduction](../orbit-sdk-introduction.md) page.
 
-### Deployment Steps
+### Deployment steps 
 
-The deployment process of AnyTrust chains is very similar to that of [Rollup chain](orbit-sdk-deploying-rollup-chain.md#rollup-config-param) but with some differences that we'll go through in this guide.
+The deployment process of AnyTrust chains is very similar to that of [Rollup chains](orbit-sdk-deploying-rollup-chain.md#rollup-config-param), but with some differences that we'll discuss in this guide.
 
 Here are the steps involved in the deployment process:
 
-1. **[Setting Up the Chain Parameters](#1-setting-up-the-chain-parameters)**
-2. **[Deploying Your AnyTrust chain](#2-deploying-your-anytrust-chain)**
-3. **Getting the AnyTrust Orbit Chain Information After Deployment**
-4. **Setting Valid Keyset on Parent Chain**
+1. **[Setting up the chain parameters](#1-setting-up-the-chain-parameters)**
+2. **[Deploying your AnyTrust chain](#2-deploying-your-anytrust-chain)**
+3. **[Getting the AnyTrust Orbit chain information after deployment](#3-getting-the-anytrust-orbit-chain-information-after-deployment)**
+4. **[Setting a valid key set on parent chain](#4-setting-valid-key-set-on-parent-chain)**
 
-The deployment of an AnyTrust Orbit chain involves defining and setting up the <a data-quicklook-from="data-availability-committee-dac">`Data Availability Committee (DAC)`</a> keyset. This keyset, comprising keys from the appointed members of the DAC, is essential for ensuring data availability and integrity. Once you have selected your committee members and gathered their keys, these keys are then configured into a keyset using the Orbit SDK. This keyset is subsequently embedded into the chain, serving as a verification mechanism to maintain the trust and security of the AnyTrust chain. The proper configuration and deployment of this keyset are vital for the effective operation and reliability of the AnyTrust model within the Arbitrum Orbit framework.
+The deployment of an AnyTrust Orbit chain involves defining and setting up the <a data-quicklook-from="data-availability-committee-dac">`Data Availability Committee (DAC)`</a> keyset. This key set includes keys from the appointed members of the DAC. They are required to ensure the chain's data availability and integrity. Once you have selected your committee members and gathered their keys, the Orbit SDK helps you configure these keys into a keyset. 
```

Comment:
flagging inconsistency between "keyset" / "key set" - if needed, we can use the [terminology guide](https://www.notion.so/arbitrum/Pattern-guide-Docs-dafaf8d1ce34433d88c1ef3b4b99c19c?pvs=4#92b5697554434a7ea09e1a2781aeda67) to align on & document consistent terminology patterns throughout our corpus